README.md

```diff
@@ -148,6 +149,59 @@ The API token to allow access to your Campfire account.
 
 For more options to set Campfire, like _ssl_, check [here](https://github.com/collectiveidea/tinder/blob/master/lib/tinder/campfire.rb#L17).
 
+### Datadog notifier
+
+This notifier sends error events to Datadog using the [Dogapi](https://github.com/DataDog/dogapi-rb) gem.
+
+#### Usage
+
+Just add the [Dogapi](https://github.com/DataDog/dogapi-rb) gem to your `Gemfile`:
+
+```ruby
+gem 'dogapi'
+```
+
+To use datadog notifier, you first need to create a `Dogapi::Client` with your datadog api and application keys, like this:
+
+```ruby
+cilent = Dogapi::Client.new(api_key, application_key)
```

Comment:
Minor typo, which can lead to confusion when configuring Datadog integration with EN. However, I'll not block the PR because of that.

```suggestion
client = Dogapi::Client.new(api_key, application_key)
```

Reply:
Oops. Thanks for merging it in. Here's the PR for the fix - https://github.com/smartinez87/exception_notification/pull/439